Sources/PluginViewController/PluginViewController.swift

```diff
@@ -87,6 +124,23 @@ final class PluginViewController: UIViewController {
         closeEventHandler?()
     }
 
+    @objc private func keyboardDidShow(_ notification: Notification) {
+        guard let userInfo = notification.userInfo else {
+            return
+        }
+        guard let endFrame = userInfo[UIResponder.keyboardFrameEndUserInfoKey] as? CGRect else {
+            return
+        }
```

Comment:
Please, combine these `guard`s into one

Reply:
But its following our <a href = "https://github.com/rosberry/swift-style-guide#golden-path">code style</a> golden path. Did you sure this change is required?

Reply:
I'm pretty sure that this looks the way elegant
```
guard let userInfo = notification.userInfo,
      let endFrame = userInfo[UIResponder.keyboardFrameEndUserInfoKey] as? CGRect else {
    return
    }
```